FabricTestExample/App.js

```diff
@@ -98,6 +99,10 @@ import Test2028 from './src/Test2028';
 import Test2048 from './src/Test2048';
 import Test2069 from './src/Test2069';
 
+// Prevent RTL tests from propagating direction to other tests.
+// Remember to reload the app after switching from the test with RTL mode.
+I18nManager.forceRTL(false);
```

Comment:
I wonder if it wouldn't be better to revert back on component unmount instead of global, what do you think?

Reply:
Yeah, great idea! Resolved in https://github.com/software-mansion/react-native-screens/pull/2130/commits/791b21be203fb8e5801c6d6f0013776c66b00179.